src/hotspot/share/runtime/continuation.hpp

```diff
@@ -39,7 +39,7 @@ class RegisterMap;
 class Continuations : public AllStatic {
 public:
   static void init();
-  static bool enabled(); // TODO: used while virtual threads are in Preview; remove when GA
+  static bool enabled();
```

Comment:
Based on the comment, I was expecting this line to be removed.
I'll take it that there's still a way to disable Continuations even after virtual
threads come out of "Preview"?

Reply:
And now I'm mulling why `VMContinuations` doesn't a default value...

Reply:
Ohhh... that's right. It's not implemented on all platforms yet... got it...

Reply:
That's right. Additionally, the alternative implementation of virtual threads can be tested by running tests with -XX:-VMContinuations, and we have a few tests that do this.